app/google-services.json

```diff
@@ -1,43 +1,35 @@
 {
   "project_info": {
-    "project_id": "sunshineapp-1293",
     "project_number": "255510533533",
-    "name": "SunshineApp"
+    "project_id": "sunshineapp-1293"
   },
   "client": [
     {
       "client_info": {
         "mobilesdk_app_id": "1:255510533533:android:83b792e26f07bf18",
-        "client_id": "android:com.example.android.sunshine.app",
-        "client_type": 1,
         "android_client_info": {
-          "package_name": "com.example.android.sunshine.app",
-          "certificate_hash": []
+          "package_name": "com.example.android.sunshine.app"
         }
       },
       "oauth_client": [],
-      "api_key": [],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBnkk0ZR716RFls1t1MBeqsNwHsilOoiLU"
+        }
+      ],
       "services": {
         "analytics_service": {
           "status": 1
         },
-        "cloud_messaging_service": {
-          "status": 2,
-          "apns_config": []
-        },
         "appinvite_service": {
           "status": 1,
           "other_platform_oauth_client": []
         },
-        "google_signin_service": {
-          "status": 1
-        },
         "ads_service": {
           "status": 1
         }
       }
     }
   ],
-  "client_info": [],
-  "ARTIFACT_VERSION": "1"
+  "configuration_version": "1"
```

Comment:
you don't mention any of these changes in the PR description (as in, why did you remove some fields)